src/inverseRobot.js

```diff
@@ -8,6 +8,29 @@
 
 function inverseRobot(robot) {
   // write code here
+  let count = 0;
+  const k = [];
+
+  for (const key in robot) {
+    count++;
+    k.push(key);
+  }
+
+  for (let i = 0; i < count; i++) {
+    for (let j = i + 1; j < count; j++) {
+      if (Object.values(robot)[i] === Object.values(robot)[j]) {
```

Comment:
```suggestion
```
I think its better to define a variable at the start for the object values, cause in this case its recreate the array every time when the loop executes.